src/middlewares/minifyAvatar.js

```diff
@@ -0,0 +1,34 @@
+const imagemin = require("imagemin");
+const imageminJpegtran = require("imagemin-jpegtran");
+const imageminPngquant = require("imagemin-pngquant");
+const path = require("path");
+const { promises: fsPromises } = require("fs");
+const { errorHandler } = require("../helpers");
+const config = require("../../config");
+
+module.exports = async function (req, res, next) {
+  try {
+    const filePathForImagemin = `E:/NodeJS/goit-nodejs-homeworks/src/tmp/${req.file.filename}`;

```

Comment:
do not use absolute path

https://www.google.ru/search?q=absolute+path+vs+relative+path&oq=absolute+path+&sourceid=chrome&ie=UTF-8